### ISSUE TITLE
Jumbo frames not working on AP mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [padavanfirmware/rt-n56u#1359]
```
What steps will reproduce the problem?
1. ping -f -l 9000 192.168.0.5
* from machine connected directly on LAN port

What is the expected output? 
Reply from 192.168.0.5: bytes=1452 time<1ms TTL=64
What do you see instead?
Packet needs to be fragmented but DF set.

What version of the product are you using? On what operating system?
Firmware:   3.0.3.8-081

Please describe the problem as detailed as it's possible.

Basically, my RT-N56U is set as an AP and Jumbo frames dont seems to work at 
all. The result of two devices connected on the LAN ports cannot send frames 
larger the 1472 bytes. By looking a bit into it, I found that only the lo 
interface is set to a MTU of 16k bytes and all other are 1500 bytes (eth2, br0, 
etc).
```

Original issue reported on code.google.com by `julioqc47` on 7 Nov 2014 at 11:20
